docs/README.md

```diff
@@ -31,7 +31,7 @@ here](https://nvidia-merlin.github.io/core).
    directory:
 
    ```sh
-   http-server build/html
+   python -m http.server -d build/html
```

Comment:
the python http.server module isn't the best imo (and I prefer using the node.js version that was previously listed here - it handles concurrent requests much better, and doesn't flake out as much at least for me).

However, If we are going to recommend going with the python http.server, maybe we can ditch the recommendation on line 37 to install the node.js one ?

Reply:
Doh!  I missed that.  I just noticed that I wasn't successful running the command so I substituted back in the vanilla Python method.

I'm not sure that my opinion matters for much, but to me, I have a preference to keeping all the dependencies for a project in a distinct venv.  Mixing in node...I guess my pref is to show the simplest way.  More knowledgeable folks like yourself don't have to follow instructions.

I'll remove line 37. Thank you.